src/DataLayer/converters/MeasureConverter.ts

```diff
@@ -22,8 +23,9 @@ function convertMeasure(measure: IMeasure): AFM.IMeasure {
 
     const convertedDefinition = convertMeasureDefinition(definition);
 
+    const requestedMeasureFormat = get(measure.measure, 'format', undefined);
```

Comment:
its necessary to specify `undefined`? `lodash.get` by default its returning `undefined`

Reply:
fixed